read_data.rb

```diff
@@ -0,0 +1,43 @@
+require 'json'
+
+def read_data
+  read_books
+  read_people
+  read_rentals
+end
+
+def read_books
+  file = File.read('books.json') if File.exist?('books.json')
+  books = JSON.parse(file) unless file.chomp.empty?
+  @my_app.books = books&.map { |book| Book.new(book['title'], book['author']) } || []
+end
+
+def read_people
+  file = File.read('people.json') if File.exist?('people.json')
+  people = JSON.parse(file) unless file.chomp.empty?
+  people&.map do |person|
+    if person['class_instance'] == 'Student'
+      student = Student.new(person['age'], person['classroom'], person['name'], person['parent_permission'])
+      student.id = person['id']
+      @my_app.people.push(student)
+    else
+      teacher = Teacher.new(person['age'], person['specialization'], person['name'])
+      teacher.id = person['id']
+      @my_app.people.push(teacher)
+    end
+  end || []
+end
+
+def read_rentals
+  file = File.read('rentals.json') if File.exist?('rentals.json')
+  rentals = JSON.parse(file) unless file.chomp.empty?
+  add_rental(rentals)
+end
```

Comment:
- [ ] You should handle the case when one or more of the JSON files are missing. I tried to remove the `people..json, retals.json, books.json` files. When I run your app, it shows the following error:

<img width="677" alt="image" src="https://user-images.githubusercontent.com/77190203/181219724-87edd4f2-b810-40f8-a934-1925c421d62c.png">

Please make sure that if one of the files is missing, your app will create a new one with an empty array and save the data there when the app is exited.